widget_driver/example/pubspec.yaml

```diff
@@ -11,7 +11,8 @@ dependencies:
   flutter:
     sdk: flutter
   cupertino_icons: ^1.0.2
-  widget_driver: ^0.0.4
+  widget_driver:
+    path: ../
```

Comment:
Same here.. All of these pubspec.yaml changes need to be reverted.. 
I just changed them so that I could run everything locally